conn.go

```diff
@@ -136,35 +136,36 @@ const (
 )
 
 const (
-	NzTypeRecAddr = 1 + iota // !NOTE-bmz need to add this to all switch stmts
-	NzTypeDouble
-	NzTypeInt
-	NzTypeFloat
-	NzTypeMoney
-	NzTypeDate
-	NzTypeNumeric
-	NzTypeTime
-	NzTypeTimestamp
-	NzTypeInterval
-	NzTypeTimeTz
-	NzTypeBool
-	NzTypeInt1
-	NzTypeBinary
-	NzTypeChar
-	NzTypeVarChar
-	NzDEPR_Text // OBSOLETE 3.0: BLAST Era Large 'text' Object
-	// (Postgres 'text' datatype overload, too)
-	NzTypeUnknown // corresponds to PG UNKNOWNOID data type - an untyped string literal
-	NzTypeInt2
-	NzTypeInt8
-	NzTypeVarFixedChar
-	NzTypeGeometry
-	NzTypeVarBinary
-	NzDEPR_Blob // OBSOLETE 3.0: BLAST Era Large 'binary' Object
-	NzTypeNChar
-	NzTypeNVarChar
-	NzDEPR_NText    // OBSOLETE 3.0: BLAST Era Large 'nchar text' Object
-	NzTypeLastEntry // KEEP THIS ENTRY LAST - used internally to size an array
+	NzTypeRecAddr      = 1 // !NOTE-bmz need to add this to all switch stmts
```

Comment:
why explicitly define every constant instead of using **iota**?

Reply:
Because auto-increment of the number would result in `NzTypeJson = 28` and `NzTypeJson = 29`, which are not the same as what we have in the backend ( `NzTypeJson = 30` and `NzTypeJson = 31`).

Reply:
It's been changed back to auto enumeration with the blank identifier `_` being used as a placeholder for the missing enums.